eq-author/README.md

```diff
@@ -136,97 +45,12 @@ Builds (via `yarn build`) and deploys the project to Github Pages.
-
-2. Runtime configurable variables. These are values that can change for each place we run the app for example in staging we want the API url to be different to production. In the code these values are read using the config object for example `config.{key}`.
-   - Dev - Values are read from the environment.
-   - Docker - Values are read from `window.config` (as defined in `index.html`) and then `process.env`. `index.html` is rewritten in docker to read the available environment variables and pass them to the application every time the docker image starts.
-
-## Testing
-
-### Integration tests
-
-Author's integration testing is run using the Cypress framework and can be run using the following commands:
-
-- `yarn test:integration`
-
-Launches Cypress on Chrome and automatically runs the default test suite.
-
-- `yarn cypress:open`
-
-Launches Cypress using the Electron framework and allows for choosing which test to run and a more interactive and detailed testing enviroment.
-
-By default the integration tests will be run against `http://localhost:13000` as configured in the [.env configuration](.env.test). It is possible to point Cypress at another environment by overriding the `CYPRESS_baseUrl` environment variable.
-
-e.g. `CYPRESS_baseUrl=http://some-other-environment yarn cypress:open`
-
-### Filename Conventions
-
-Tests are colocated next to the code they are testing. For example, a test for `/src/components/Button/index.js` could be in a file `/src/components/Button/test.js`.
-
-### Command Line Interface
-
-When you run `yarn test`, Jest will launch in the watch mode. Every time you save a file, it will re-run the tests, just like `yarn start` recompiles the code.
-
-The watcher includes an interactive command-line interface with the ability to run all tests, or focus on a search pattern. It is designed this way so that you can keep it open and enjoy fast re-runs.
-
-## Troubleshooting
-
-### Jest crashing
-
-#### Problem
-
-Running `yarn test` causes Jest to crash with the following error:
-
-```
-(FSEvents.framework) FSEventStreamStart: register_with_server: ERROR: f2d_register_rpc() => (null) (-22)
-(FSEvents.framework) FSEventStreamStart: register_with_server: ERROR: f2d_register_rpc() => (null) (-22)
-(FSEvents.framework) FSEventStreamStart: register_with_server: ERROR: f2d_register_rpc() => (null) (-22)
-events.js:160
-      throw er; // Unhandled 'error' event
-      ^
-
-Error: Error watching file for changes: EMFILE
-    at exports._errnoException (util.js:1036:11)
-    at FSEvent.FSWatcher._handle.onchange (fs.js:1406:11)
-```
-
-#### Solution
+1.  Build time environment variables. These are values that are known at build time and cannot be changed once the docker image is built. Currently these are only `NODE_ENV` and `REACT_APP_AUTH_ENABLED`. These are referenced in the code as `process.env.{key}`
```

Comment:
`REACT_APP_AUTH_ENABLED` doesn't seem to do anything any more so this can be removed.

Reply:
I have removed this section now.